cd/updateCopyrightHeaders.sh

```diff
@@ -10,14 +10,14 @@ do
     for startYear in `seq 2016 1 $((currentYear - 1))`;
     do
         sedCmd="s/Copyright ${startYear} Crown Copyright/Copyright ${startYear}-${currentYear} Crown Copyright/g"
-        #echo "sed -i '' ${sedCmd} ${f}"
-        sed -i '' "$sedCmd" ${f}
+        #echo "sed -i'' -e "$sedCmd" ${f}"
+        sed -i'' -e "$sedCmd" ${f}
```

Comment:
Just tried the script and it leaves lots of temporary files due to the -e argument.